packages/evasive-transform/src/parse-ast.js

```diff
@@ -38,6 +36,7 @@ export function parseAst(source, opts = {}) {
   return parseBabel(source, {
     tokens: true,
     createParenthesizedExpressions: true,
+    allowReturnOutsideFunction: opts.sourceType === 'script',
```

Comment:
```suggestion
    allowReturnOutsideFunction: opts.sourceType !== 'module',
```
^ this would cover cases of sourceType not being set/known

Reply:
paraphrasing @kriskowal, it is only for `script` because we do not want to be toxic enablers of non-portable syntax